secretstores/feature.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package secretstores
+
+const (
+	// FeatureMultipleKeyValuesPerSecret advertises that this SecretStore supports multiple keys-values under a single secret.
+	FeatureMultipleKeyValuesPerSecret Feature = "MULTIPLE_KEY_VALUES_PER_SECRET"
+)
+
+// Feature names a feature that can be implemented by Secret Store components.
+type Feature string
```

Comment:
nit: invert the declaring order: Define type feature first then define by its values below

Reply:
I was following the example from [state/feature.go](https://github.com/dapr/components-contrib/blob/2b5650c2faa41f0a8d82862583497103776be49f/state/feature.go) and [pubsub/feature.go](https://github.com/dapr/components-contrib/blob/2b5650c2faa41f0a8d82862583497103776be49f/pubsub/feature.go) but yeah, you are right, it would look and read better inverted. Fixed in next revision.